### PR TITLE
Sending all kick methods

### LINKED ISSUE #1281: [Request]: Choose Kick in Reactions
### Problem

_No response_

### Solution

Add a List of Kick's after checking the box "Kick Player", 
Make it that you can select only one.
List : 
Null Function Kick.
End Session Kick.
Desync Kick.
OOM Kick.
Bail Kick.

### Reason

I sometimes wanna use others kicks in reactions

### Additional context

Oh and If a kick Doesn't Work Out then Use the other one

### LINKED ISSUE #1443: [Bug]: Kick reaction to an event
### Describe the bug

As far as I know, breakup kick was fixed with the latest update, maybe you should use desync kick as a reaction, since the yimmenu cannot block it, for example, if another cheater tries to kick you or leave the function of choosing how to react to what the cheater does to you. It’s just that I haven’t met a cheater yet who could be kicked by me using the bail kick method. Maybe similar to #1281 Although, according to the idea for the host, you can choose another way to kick

### Steps To Reproduce

1. Open settings/reactions
2. Add a pop-up list of kick methods there after the "player kick" checkbox is active, it is possible to leave the desync kick by default and choose some other method for the cheater host

### Expected Behavior

The cheater should be automatically kicked when he tries to crash or kick me, etc. Because bail kick does not work on most cheaters or at least add the ability to select the kick method. 

### Screenshots and/or Logs

```
<paste your logs here>
```

### Store

Steam

### Additional context

_No response_

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### LINKED ISSUE #1833: [Bug]: squad weapons bug
### Describe the bug

When I want to change the weapon to any other one in the saved template (an example of a flare gun in the screenshot), the squads still continue to shoot from the plane with an uzi. The same situation, for example, if I spawn a squad on a motorcycle with a small hand grenade launcher, then he still has an uzi. I have not tried other weapons, but I think the situation will be the same. Settings below (maybe I did something wrong)
pic

### Steps To Reproduce

1. Create a squad in luxor and give him a flare gun
2. Watch him shoot you or someone else with an uzi

### Expected Behavior

Squads must fire their flare pistol. And even when you've already been killed once

### Logs and/or screenshots

```
<paste your logs here>
```

### Store

Steam

### Additional context

_No response_

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

### PR DESCRIPTION
**Pull request by ghost**
It seems to me that there is a much better chance of kicking a player without being a host. Closes #1443 closes #1281 closes #1833

### CODE REVIEW

**Comment by rrAsus** [comment]
and #1281 ?

**Comment by ghost** [comment]
> and #1281 ?

Exactly. Why choose any kick method when you can send all methods

**Comment by rrAsus** [comment]
> > and #1281 ?
> 
> Exactly. Why choose any kick method when you can send all methods

you mean all-in-one kick?

**Comment by ghost** [comment]
> > > and #1281 ?
> > 
> > 
> > Exactly. Why choose any kick method when you can send all methods
> 
> you mean all-in-one kick?

Yes, just send all the kick methods as a reaction. I don't know if I can make a multi-hit button just I haven't worked with gui but I can try. It's still better than sending different methods separately

**Comment by rrAsus** [comment]
> > > > and #1281 ?
> > > 
> > > 
> > > Exactly. Why choose any kick method when you can send all methods
> > 
> > 
> > you mean all-in-one kick?
> 
> Yes, just send all the kick methods as a reaction. I don't know if I can make a multi-hit button just I haven't worked with gui but I can try. It's still better than sending different methods separately

Best To Include Both. Seperate Method's & All-In-One.

**Comment by ghost** [comment]
Guys, I have a question, what am I doing wrong when I try to add a multi-kick command? I forgot to add it somewhere, can you tell me where? Because of this, my command does not exist and the button does not appear. Sorry, I just want to understand a little how it works.
png

**Comment by sch-lda** [comment]
since you are using script::get_current()->yield(700ms);, you should add #include "script.hpp" in multi_kick.cpp.

**Comment by ghost** [comment]
> since you are using script::get_current()->yield(700ms);, you should add #include "script.hpp" in multi_kick.cpp.

Thanks bro, everything seems to be working now. Waiting for the review

**Comment by maybegreat48** [comment]
The bail kick is hotfixed by Rockstar and should be removed

File: src/services/context_menu/context_menu_service.hpp [hunk 1]
```diff
@@ -192,10 +192,12 @@ namespace big
 			            static player_command* command2 = dynamic_cast<player_command*>(command::get(rage::consteval_joaat("endkick")));
 			            static player_command* command3 = dynamic_cast<player_command*>(command::get(rage::consteval_joaat("desync")));
 			            static player_command* command4 = dynamic_cast<player_command*>(command::get(rage::consteval_joaat("breakup")));
+				    static player_command* command5 = dynamic_cast<player_command*>(command::get(rage::consteval_joaat("multikick")));
 			            command->call(ped::get_player_from_ped(m_handle), {});
```
Comment (maybegreat48) [inline src/services/context_menu/context_menu_service.hpp:191]: You don't need to call the other kicks anymore

**Comment by gir489returns** [comment]
> The bail kick is hotfixed by Rockstar and should be removed

When was this?

**Comment by egssy0** [comment]
> The bail kick is hotfixed by Rockstar and should be removed

lol no its still possible to do but there's a method for stopping it

**Comment by ghost** [comment]
> > The bail kick is hotfixed by Rockstar and should be removed
> 
> lol no its still possible to do but there's a method for stopping it

well I tried and this kick doesn't work

**Comment by maybegreat48** [comment]
> > The bail kick is hotfixed by Rockstar and should be removed
> 
> When was this?

Probably a week ago

**Review by Yimura — APPROVED** [review]

**Pull request merged**